next.config.js

```diff
@@ -6,5 +6,12 @@ const withMDX = require('./lib/mdx-loader')({
 })
 
 module.exports = withMDX({
-  pageExtensions: ['md', 'mdx', 'jsx', 'js']
+  pageExtensions: ['md', 'mdx', 'jsx', 'js'],
+  webpack: config => {
+    return Object.assign({}, config, {
+      node: {
+        fs: 'empty'
+      }
+    })
```

Comment:
Kenapa kita butuh ini?

Reply:
Kyknya ga butuh sih, told you banyak rubbish 🙈

Reply:
Never mind, take it easy